### PR TITLE
addressed islandora-2252

### PR DESCRIPTION
**Pull request by ajstanley**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2252)

# What does this Pull Request do?

Clarifies installation documnetation

# What's new?
Changes a few installation messages, uses variable if set.

# How should this be tested?

If the solution pack is installed on a server with a misconfigured kdu_compress the new messages should appear.

# Additional Notes:
None

# Interested parties
@DiegoPino

### CODE REVIEW

File: islandora_large_image.install
Comment (DiegoPino) [inline islandora_large_image.install]: Hi, just a tiny issue here. $kdu_compress needs to be escaped for unix and windoes or made "safe" -somehow-here. I know not a big vulnerability but this can lead easily to arbitrary code execution. Maybe just via http://www.php.net/escapeshellarg. @Islandora/7-x-1-x-committers anyone with a simple to follow example here?

**Review by DiegoPino — CHANGES_REQUESTED** [review]
escaping shell arguments can help secure that command php

**Comment by ajstanley** [comment]
@DiegoPino - looks okay now?

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
@ajstanley looks good now, thanks!. @Islandora/7-x-1-x-committers will wait 24 hours before merging, feel free to re-review before i press the green button.

**Comment by DonRichards** [comment]
@DiegoPino Ready to merge?

**Pull request merged**
